src/components/editor/Store/hooks.ts

```diff
@@ -533,6 +533,7 @@ export const useEditorStore_queryResponse_mutate = (
 
 export const useEditorStore_queryResponse_draftedBindingIndex = (
     collection: string | null,
+    targetBindingIndex: number,
     params?: SelectorParams | undefined
 ) => {
     const localScope = params?.localScope;
```

Comment:
With where this is getting called I do not think we need to support the `localScope` stuff anymore.